include/dlaf/multiplication/general/api.h

```diff
@@ -40,5 +40,28 @@ DLAF_MULTIPLICATION_GENERAL_ETI(extern, Backend::GPU, Device::GPU, double)
 DLAF_MULTIPLICATION_GENERAL_ETI(extern, Backend::GPU, Device::GPU, std::complex<float>)
 DLAF_MULTIPLICATION_GENERAL_ETI(extern, Backend::GPU, Device::GPU, std::complex<double>)
 #endif
+
+template <Backend B, Device D, class T>
+struct GeneralSubK {
```

Comment:
If the main algorithm is renamed, this should probably also be renamed to something consistent.